medusa/search/proper.py

```diff
@@ -201,53 +200,54 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
                 if cur_proper.name not in processed_propers_names:
                     self.processed_propers.append({'name': cur_proper.name, 'date': cur_proper.date})
                 continue
 
-            if not parse_result.episode_numbers:
+            if not cur_proper.parse_result.episode_numbers:
                 logger.log('Ignoring full season instead of episode: {name}'.format
                            (name=cur_proper.name), logger.DEBUG)
                 if cur_proper.name not in processed_propers_names:
                     self.processed_propers.append({'name': cur_proper.name, 'date': cur_proper.date})
                 continue
 
             logger.log('Successful match! Matched {original_name} to show {new_name}'.format
-                       (original_name=parse_result.original_name, new_name=parse_result.show.name), logger.DEBUG)
+                       (original_name=cur_proper.parse_result.original_name, new_name=cur_proper.parse_result.show.name), logger.DEBUG)
 
-            # set the indexerid in the db to the show's indexerid
-            cur_proper.indexerid = parse_result.show.indexerid
+            # Map the indexerid in the db to the show's indexerid
+            cur_proper.indexerid = cur_proper.parse_result.show.indexerid
 
-            # set the indexer in the db to the show's indexer
-            cur_proper.indexer = parse_result.show.indexer
+            # Map the indexer in the db to the show's indexer
+            cur_proper.indexer = cur_proper.parse_result.show.indexer
 
-            # populate our Proper instance
-            cur_proper.show = parse_result.show
-            cur_proper.season = parse_result.season_number if parse_result.season_number is not None else 1
-            cur_proper.episode = parse_result.episode_numbers[0]
-            cur_proper.release_group = parse_result.release_group
-            cur_proper.version = parse_result.version
-            cur_proper.quality = parse_result.quality
+            # Map our Proper instance
+            cur_proper.show = cur_proper.parse_result.show
+            cur_proper.actual_season = cur_proper.parse_result.season_number if cur_proper.parse_result.season_number is not None else 1
+            cur_proper.actual_episodes = cur_proper.parse_result.episode_numbers
+            cur_proper.release_group = cur_proper.parse_result.release_group
+            cur_proper.version = cur_proper.parse_result.version
+            cur_proper.quality = cur_proper.parse_result.quality
             cur_proper.content = None
-            cur_proper.proper_tags = parse_result.proper_tags
+            cur_proper.proper_tags = cur_proper.parse_result.proper_tags
+
+            # filter release, in this case, it's just a quality gate. As we only send one result.
+            best_result = pick_best_result(cur_proper)
 
-            # filter release
-            best_result = pick_best_result(cur_proper, parse_result.show)
```

Comment:
@p0psicles can we change the log message below this code `Rejected proper due to release filters` be changed to like `Rejected proper:` as we log the reason why it was rejected in pick_best_result and its not only because of release filters. can be rejected because of quality too